app/client/components/DeleteModeButton.js

```diff
@@ -0,0 +1,33 @@
+
+import React from 'react';
+import Glyphicon from 'react-bootstrap/lib/Glyphicon';
+
+export default class DeleteModeButton extends React.Component {
+
+    static propTypes = {
+        isActive: React.PropTypes.bool,
+        onClick: React.PropTypes.func,
+    }
+
+    render() {
+        var { isActive } = this.props;
+        var { handleChange } = this;
+
+        var deleteMode = isActive ? 'shake-little' : 'dont-shake';
```

Comment:
Can we improve this variable name?
It is specifically used for a CSS class name.